libclamav/others_common.c

```diff
@@ -176,18 +177,31 @@ inline void cli_dbgmsg(const char *str, ...)
 {
     if (UNLIKELY(cli_get_debug_flag())) {
         MSGCODE(buff, len, "LibClamAV debug: ");
-        fputs(buff, stderr);
+        clrs_eprint(buff);
     }
 }
 
 void cli_dbgmsg_no_inline(const char *str, ...)
 {
     if (UNLIKELY(cli_get_debug_flag())) {
         MSGCODE(buff, len, "LibClamAV debug: ");
-        fputs(buff, stderr);
+        clrs_eprint(buff);
     }
 }
 
+size_t cli_eprintf(const char *str, ...)
+{
+    va_list args;
+    char buff[MSGBUFSIZ];
+    va_start(args, str);
+    vsnprintf(buff, sizeof(buff), str, args);
+    buff[sizeof(buff) - 1] = '\0';
+    va_end(args);
+    clrs_eprint(buff);
```

Comment:
Any mechanism to return errors?

Reply:
I have `clrs_eprint()` printing lossy, so that invalid utf-8 graphemes render inline with the rest of a message rather than failing to log it entirely.  For `stderr` logg messages I think that makes sense. 

Personally I'm okay with that.  If you pass in a NULL pointer, having it print nothing and not report an error also seems fine to me.  It's not like anyone is going to put error handling on a log function anyways.

Reply:
S'okay.  Had to say something.